WordPress/src/main/java/org/wordpress/android/ui/blaze/ui/blazeoverlay/BlazeOverlayFragment.kt

```diff
@@ -0,0 +1,282 @@
+            else -> null
+        }
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier.padding(
+                top = Margin.ExtraLarge.value,
+                start = Margin.ExtraLarge.value,
+                end = Margin.ExtraLarge.value
+            )
+        ) {
+            Image(
+                painterResource(id = R.drawable.ic_blaze_promotional_image),
+                contentDescription = stringResource(id = R.string.blaze_activity_title),
+                modifier = Modifier.size(100.dp)
+            )
+            Text(
+                stringResource(id = R.string.blaze_promotional_text),
+                fontSize = FontSize.DoubleExtraLarge.value,
+                fontWeight = FontWeight.Bold,
+                modifier = Modifier.padding(top = Margin.ExtraExtraMediumLarge.value),
+                textAlign = TextAlign.Center
+            )
+            Subtitles(
+                listOf(
+                    R.string.blaze_promotional_subtitle_1,
+                    R.string.blaze_promotional_subtitle_2,
+                    R.string.blaze_promotional_subtitle_3
+                ),
+                modifier = Modifier.padding(start = 20.dp, end = 20.dp)
+            )
+            post?.let {
+                PostThumbnailView(
+                    it, modifier = Modifier
+                        .fillMaxWidth()
+                        .wrapContentHeight()
+                )
+            }
+
+            ImageButton(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(48.dp)
+                    .padding(start = 20.dp, end = 20.dp)
+                    .background(
+                        Color.Black,
+                        shape = RoundedCornerShape(15.dp)
+                    ),
+                buttonText = UiString.UiStringRes(R.string.blaze_post_promotional_button),
+                drawableRight = Drawable(R.drawable.ic_promote_with_blaze),
+                onClick = { viewModel.onPromoteWithBlazeClicked() }
+            )
+        }
+    }
+
+    @Preview
+    @Composable
+    fun PreviewThumbNailView() {
+        AppTheme {
+            BlazeOverlayContent(
+                postModelState = BlazeUiState.PromoteScreen.PromotePost(
+                    PostUIModel(
+                        postId = 119,
+                        title = "Post title check if this is long enough to be truncated",
+                        url = "https://www.google.long.ttiiitititititiit.com",
+                        imageUrl = 0
+                    )
+                )
+            )
+        }
+    }
+
+    @Composable
+    fun PostThumbnailView(postUIModel: PostUIModel, modifier: Modifier = Modifier) {
+        ConstraintLayout(
+            modifier = modifier
+                .padding(start = 20.dp, end = 20.dp, top = 15.dp, bottom = 15.dp)
```

Comment:
np: You can use `padding(horizontal = 20.dp, vertical = 15.dp)` in such cases

Reply:
Thanks, TIL. I have done the suggested change in 0cf5f9fb6ddc0d945b193537984aa76a74a37c68. 

🗒️  Ignore the commit of the .idea/checkstyle-idea.xml in the above commit. I have reverted that in another commit d82fdda3713ef38cd85c96f0a01bcc63c48377c2. Env issues.